From: Quan Nguyen <qnguyen@apm.com>
Subject: arm64: dts: Update X-Gene standby GPIO controller DTS entries
Patch-mainline: v4.6-rc1
Git-commit: 47f134a2d585b9bee251d5ae8d671e5e2cd93247

Update APM X-Gene standby GPIO controller DTS entries to enable it
as interrupt controller.

Signed-off-by: Y Vo <yvo@apm.com>
Signed-off-by: Quan Nguyen <qnguyen@apm.com>
Reviewed-by: Linus Walleij <linus.walleij@linaro.org>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>

---
arch/arm64/boot/dts/apm/apm-storm.dtsi | 3 +++
 arch/arm64/boot/dts/apm/apm-storm.dtsi |    3 +++
 1 file changed, 3 insertions(+)

--- a/arch/arm64/boot/dts/apm/apm-storm.dtsi
+++ b/arch/arm64/boot/dts/apm/apm-storm.dtsi
@@ -771,6 +771,9 @@
 					<0x0 0x2b 0x1>,
 					<0x0 0x2c 0x1>,
 					<0x0 0x2d 0x1>;
+			interrupt-parent = <&gic>;
+			#interrupt-cells = <2>;
+			interrupt-controller;
 		};
 
 		rtc: rtc@10510000 {
